R/create_mofa_from_mae.R

```diff
@@ -76,31 +114,102 @@ setMethod("mofa2", signature = c(mae = "MultiAssayExperiment"),
 
 ########################## HELP FUNCTIONS ##########################
 
+# Select experiments from MAE
+#' @importFrom MultiAssayExperiment experiments
+.select_experiments <- function(mae, experiments) {
+  # Check that the value is correct
+  is_name <- is.character(experiments) && length(experiments) > 0 &&
+    length(experiments) <= length(experiments(mae)) &&
+    all(experiments %in% names(experiments(mae)))
```

Comment:
Probably `names(experiments(mae))` can be simplified to `names(mae)` here and elsewhere.